htmresearch/regions/TemporalPoolerRegion.py

```diff
@@ -223,9 +242,9 @@ def __init__(self, columnCount, inputWidth, poolerType, **kwargs):
     # These calls whittle down kwargs and create instance variables of TemporalPoolerRegion
     self._poolerClass = _getPoolerClass(poolerType)
     pArgTuples = _buildArgs(self._poolerClass, self, kwargs)
-
+    pArgTuplesSP = _buildArgs(_getParentSpatialPoolerClass(), self, kwargs)
```

Comment:
I changed _getParentSpatialPoolerClass to be more general. It will always return the spatial pooler parent of the poolerClass in TPRegion